jest.config.js

```diff
@@ -1,3 +1,5 @@
+process.env.TZ = 'UTC';
```

Comment:
why is this one needed? for executing `jest` directly in terminal?

Reply:
Or for editor plugins that might use Jest as well. Basically, if you're reading the configuration, you're probably running Jest—so, set this appropriately.

I'm thinking that Cypress is a lot better at mocking time and maybe it makes more sense to test this functionality in Cypress.